lib/OpenQA/App.pm

```diff
@@ -2,12 +2,11 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 package OpenQA::App;
-use Mojo::Base -strict;
+use Mojo::Base -strict, -signatures;
 
 my $SINGLETON;
 
-sub set_singleton {
-    my ($class, $app) = @_;
+sub set_singleton ($class, $app) {
```

Comment:
This function is now short enough to be put in one line.

Reply:
Okay, I'll do that